loops-and-arrays.js

```diff
@@ -5,6 +5,11 @@
 // ==========================================
 
 const names = ["Henk", "Piet", "Fred", "Joop"];
+    for (let i = 0; i < names.length; i++) {
+    names[i] = names[i] + "je"
+}
+console.log(names)
```

Comment:
Deze regel had eigenlijk ook voor de for-loop moeten staan, om de originele waardes weer te geven.